internal/provider/trigger_resource_test.go

```diff
@@ -57,7 +57,7 @@ func TestAcc_TriggerResourceUpgradeFromVersion014(t *testing.T) {
 			{
 				ExternalProviders: map[string]resource.ExternalProvider{
 					"honeycombio": {
-						VersionConstraint: "~> 0.14",
+						VersionConstraint: "~> 0.14.0",
```

Comment:
This was incorrectly testing the upgrade (and pulling in 0.15.2 at the time of the writing). This now properly tests the upgrade from the SDK-based resource